pkg/labels/labels.go

```diff
@@ -0,0 +1,13 @@
+package labels
+
+import (
+	"github.com/kubernetes-sigs/kernel-module-management/internal/utils"
+)
+
+func GetKernelModuleReadyNodeLabel(namespace, moduleName string) string {
+	return utils.GetKernelModuleReadyNodeLabel(namespace, moduleName)
+}
+
+func GetDevicePluginNodeLabel(namespace, moduleName string) string {
+	return utils.GetDevicePluginNodeLabel(namespace, moduleName)
+}
```

Comment:
Could we move the existing functions instead of defining new ones?

Reply:
I would prefer to keep it like that: currently we have all the code defining the labels in the internal/utils/kmmlabels.go which is convinient. If we move public labels to pkg/label it would work but it will split the code and we will have 2 package handling the labels. For now i would prefer to keep the things as is. We can get back to this discussion in the nig refactoring task that we have scheduled for 2.1